src/Chirp.CSVDBService/Program.cs

```diff
@@ -0,0 +1,11 @@
+using Chirp.CSVDBService;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+var database = CheepDatabase.Instance;
+
+app.MapPost("/cheep", (Cheep cheep) => database.Store(cheep));
+app.MapGet("/cheeps", () => database.Read());
```

Comment:
Is it still possible to read using a limit from the database, or will this get reintroduced in a later refactor?

Reply:
Still possible from the CheepDatabase, not possible from the Web application (at least not yet)